Rename "Custom Command" to "Shell Command"

stefanhaller opened the pull request:
- **PR Description**

The double use of the term "Custom Command" for both shell commands and user-configured keybindings was confusing.

- **Please check if the PR fulfills these requirements**

* [x] Cheatsheets are up-to-date (run `go generate ./...`)
* [x] Code has been formatted (see [here](https://github.com/jesseduffield/lazygit/blob/master/CONTRIBUTING.md#code-formatting))
* [ ] Tests have been added/updated (see [here](https://github.com/jesseduffield/lazygit/blob/master/pkg/integration/README.md) for the integration test guide)
* [x] Text is internationalised (see [here](https://github.com/jesseduffield/lazygit/blob/master/CONTRIBUTING.md#internationalisation))
* [x] Docs have been updated if necessary
* [x] You've read through your own file changes for silly mistakes etc

jesseduffield reviewed (approved):
I like it

stefanhaller commented on pkg/gui/controllers/global_controller.go:
```diff
@@ -23,9 +23,9 @@ func (self *GlobalController) GetKeybindings(opts types.KeybindingsOpts) []*type
 	return []*types.Binding{
 		{
 			Key:         opts.GetKey(opts.Config.Universal.ExecuteCustomCommand),
```
We leave the name of the keybinding unchanged, which is a bit unfortunate, but we don't want to risk breaking user's configs.

yam-liu replied:
Can we have different keys in the YAML file and the struct? This way, only the name in config.yml will remain unchanged.
We don't break the user's config, the key name in code is the new one.

stefanhaller replied:
Yes, we can do that, but it's actually the name in user's config files that bothers me more than the name in the code, so this wouldn't help with that. Still, here's a fixup that does this, at least it makes the integration tests a little less confusing: 08803bf85ebb

yam-liu replied:
What I can think of is that we can use a new key and mark the original one as deprecated. Update doc to guide new users to use the one (give users some time to do migration also). 
When time is proper for a major version release, we can delete it together with some old keys like `editCommand` `openCommand` etc.

stefanhaller replied:
Ah no, actually we can do better than that. I forgot that we have this mechanism in place already that allows migrating the user's config files when we rename things. See 832789082843.

(We couldn't use this mechanism for editCommand etc, because in that case the format of the configs also changed, so we couldn't just take them over 1-to-1. But for cases like this where we just rename the key, it's perfect.)

stefanhaller commented:
@jesseduffield Not sure what label to assign to this. It's not really an enhancement, is it? What else?

stefanhaller commented:
@jesseduffield I squashed the fixups, I'm quite happy with this now, if you want to have another look. The first commit was also added after you approved the PR. Also, the label question is still open.

jesseduffield reviewed (approved):
LGTM.

As for the label: it's indeed an interesting case. I'd just call it an enhancement.
